src/components/signin/components/Signin.js

```diff
@@ -62,12 +66,10 @@ class Signin extends React.Component {
                 {
                     signinStatus !== undefined &&
                     <span>
-                        {
-                            signinStatus ? message.success(signinMessage, 5) :
-                                message.error(signinMessage, 5)
-                        }
+                        { notifySigninStatus() + resetSignState() }
```

Comment:
trigger the `resetSignState` inside of the `notifySigninStatus` function. also never do
```
funcA() + funcB()
```
That `+` operator is doing more than you think

Reply:
```{ notifySigninStatus() && resetSignState() }``` will work but breaks the code.